lib/spring/test/acceptance_test.rb

```diff
@@ -104,15 +104,14 @@ def without_gem(name)
         end
       end
 
-      test "code changes in pre-referenced app files are picked up" do
```

Comment:
The old test was kind of obsolete. The old test checked that the changed file will be reloaded. The new behavior is that spring will restart in this case. The old test would fail because the spring restart would not result in the expected speedup.